content/en/news/releases/1.17.x/announcing-1.17/change-notes/index.md

```diff
@@ -140,6 +140,8 @@ These notices describe functionality that will be removed in a future release ac
 
 - **Fixed** `istioctl proxy-config endpoint -f -` returns `Error: open -: no such file or directory`. [Issue #43045](https://github.com/istio/istio/issues/43045)
 
+- Upgraded `client-go` to v1.26.1, removing support for `azure` and `gcp` auth plugins. [PR #43101](https://github.com/istio/istio/pull/43101).
```

Comment:
```suggestion
- **Updated** `client-go` to `v1.26.1`, removing support for `azure` and `gcp` auth plugins. [PR #43101](https://github.com/istio/istio/pull/43101)
```

Reply:
Nate that this should be moved up to be with the other **Updated** item.

Would this also be something we want on the Upgrade page? It's not so much of an Istio compatibility issue, but would be something that a user of a prior version may experience moving to 1.17.

Reply:
Done